src/components/Formore.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react-dom'
+import { Title, Section } from '../styles/Main'
+import { IconsWrapper, Icon } from '../styles/ForMore'
+import Github from '../icons/github.svg'
+import Linkedin from '../icons/linkedin.svg'
+
+const Formore = () => {
+  return (
+    <Section>
+      <Title>For more</Title>
+      <IconsWrapper>
+        <a href="https://github.com/Kristin-Larsson"><Icon src={Github} alt="Github" /></a>
```

Comment:
I would change these to a .svg-file, like the header component to make them more crisper 👌👌